clients/src/main/java/org/apache/kafka/common/security/oauthbearer/internals/secured/HttpAccessTokenRetriever.java

```diff
@@ -346,10 +350,17 @@ static String parseAccessToken(String responseBody) throws IOException {
         return sanitizeString("the token endpoint response's access_token JSON attribute", accessTokenNode.textValue());
     }
 
-    static String formatAuthorizationHeader(String clientId, String clientSecret) {
+    static String formatAuthorizationHeader(String clientId, String clientSecret, boolean urlencode) throws
```

Comment:
Instead of passing in the `urlencode` parameter here, can we use the `urlencodeHeader` directly?

Reply:
Oh, I see, this is a `static` method. I don't remember why that was necessary 🤷‍♂️